src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,37 +1,111 @@
-import { useState } from 'react';
+import React, { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
-  // Increase the count after successful form submission
-  // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+type Props = {
+  onAdd: (movie: Movie) => void;
+};
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
+  const [count, setCount] = useState(0);
+  const [movieTitle, setMovieTitle] = useState(''); // Перейменування змінної title
+  const [description, setDescription] = useState('');
+  const [imgUrl, setImgUrl] = useState('');
+  const [imdbUrl, setImdbUrl] = useState('');
+  const [imdbId, setImdbId] = useState('');
```

Comment:
Create only one state for movie
```suggestion
  const [movie, setMovie] = useState({
    title: ''.
    ....
  });
```

Reply:
Probably now I got one problem with my button!
Can u explain how to solve it?

Thanks!

<img width="1196" alt="Screenshot 2024-06-22 at 10 21 18 AM" src="https://github.com/mate-academy/react_movies-list-add-form/assets/131563162/de583636-83b4-4aea-97e2-b24e9f230a4c">